ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh

```diff
@@ -534,22 +534,23 @@ function upgrade() {
 # Monitor the upgrade status
 function check_upgrade_status() {
     local wait_upgrade="${TIMEOUT}" out avail progress
+    echo "Starting the upgrade checking on $(date "+%F %T")"
     while (( wait_upgrade > 0 )); do
         sleep 5m
         (( wait_upgrade -= 5 ))
```

Comment:
It seems the script will exit here due to we set `set -o errexit`, and so no upgrade timeout info output.

Reply:
From the [log](https://gcsweb-qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/qe-private-deck/logs/periodic-ci-openshift-openshift-tests-private-release-4.11-amd64-nightly-4.11-upgrade-from-stable-4.10-from-stable-4.9-aws-ipi-f28/1712822269610299392/artifacts/aws-ipi-f28/cucushift-upgrade-toimage/build-log.txt), I think  it is because of https://github.com/openshift/release/pull/44866/files#diff-a71e3444f4cc5e3d7ecb8ebd36fa8bd1fda181a0806d09523b318eadffaebc71L551, so I updated it to `[[ ]]`

Reply:
ah, verify the line and the following 551 line on my local, both of them would lead an exit, let me fix it together.

Reply:
Run again, you are right, the line should be the key, let me fix it.

Reply:
fixed.